spot_wrapper/spot_dance.py

```diff
@@ -155,6 +158,78 @@ def choreography_log_to_animation_file(
                 "",
             )
 
+    def stop_choreography(self) -> Tuple[bool, str, ExecuteChoreographyResponse]:
+        """
+        Issue an empty choreography sequence to stop the robot from dancing
+        """
+        CHOREO_NAME = "stop_seq"
```

Comment:
The name is used twice - once in the sequence definition and once in the execute call.

I'd rather hard code since this is a bit of a hack to compensate for a hole in the boston dynamics API. It's not actually a stop sequence or call, just playing an empty sequence that ends instantly. Hardcoding leads more cleanly into a future where there's an actual stop choreography api call